terraform/module-ecs-services/variables.tf

```diff
@@ -2,20 +2,26 @@ variable "stack_name" {}
 variable "name_prefix" {}
 variable "environment" {}
 
+variable "docker_registry" {}
```

Comment:
We need types and descriptions on these variables and defaults if appropriate

Reply:
Types and descriptions now added to all variables. Also rearranged into categories.